src/Loggers/LogText.jl

```diff
@@ -0,0 +1,41 @@
+"""
+    log_text(logger, name, text, step)
+
+Logs text with name `name` at step `step`
+"""
+function log_text(logger::TBLogger, name::String, text::String; step=nothing)
+    summ    = SummaryCollection()
+    push!(summ.value, text_summary(name, text))
+    write_event(logger.file, make_event(logger, summ, step=step))
+end
+
+function text_summary(name::String, text::String)
+    texttensorshape = TensorShapeProto(dim = Vector([TensorShapeProto_Dim(size = 1)]))
+    textstringval = Vector{Array{UInt8, 1}}()
+    push!(textstringval, Array{UInt8, 1}(text))
+    pb = PipeBuffer()
+    writeproto(pb, TextPluginData(version = 0))
+    textcontent = pb.data
```

Comment:
While you are at it, could you also define a method in `utils.jl`
```
function serialize_proto(data::Union{ProtoType, ProtoEnum})
     pb = PipeBuffer()
     writeproto(pb, data)
    pb.data
end
```

and use that? It will be used with practically every plugin and would make the code a bit easier to read.